triplet/eval_metrics.py

```diff
@@ -0,0 +1,28 @@
+"""Utility methods for computing evaluating metrics. All methods assumes greater
+scores for better matches, and assumes label == 1 means match.
+
+Extracted from:
+https://github.com/hanxf/matchnet/blob/master/eval_metrics.py
+"""
+import operator
+
+
+def ErrorRateAt95Recall(labels, scores):
+    recall_point = 0.95
+    # Sort label-score tuples by the score in descending order.
+    sorted_scores = zip(labels, scores)
+    sorted_scores.sort(key=operator.itemgetter(1), reverse=False)
```

Comment:
This line does not work for python3, as zip now returns an iterator.
see here : https://stackoverflow.com/questions/43698531/sorting-a-zipped-object-in-python-3

So i suppose it's better to either have 
```python
sorted_scores = list(zip(labels, scores))
sorted_scores.sort(key=operator.itemgetter(1), reverse=False)
```
or (the best IMO)
```python
sorted_scores = zip(labels, scores)
sorted_scores = sorted(sorted_scores, key=operator.itemgetter(1), reverse=False)
```